src/html/html-script-tag.ts

```diff
@@ -45,7 +53,19 @@ export class ScannedScriptTagImport extends ScannedImport {
     if (scannedDocument) {
       const importedDocument =
           new Document(scannedDocument, document._analysisContext);
-      importedDocument._addFeature(document);
+      // Connect the javascript document defined/referenced by the script tag to
+      // the html document which contained the script tag with a synthetic
```

Comment:
And a bit about why, maybe including an example of importing polymer.html, then referencing Polymer()